deploy/helm/sumologic/templates/metrics/collector/otelcol/opentelemetrycollector.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if and (eq (include "metrics.otelcol.enabled" .) "true") .Values.sumologic.metrics.collector.otelcol.enabled }}
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: {{ template "sumologic.metadata.name.metrics.collector.opentelemetrycollector" . }}
+  labels:
+    {{- include "sumologic.labels.metrics.opentelemetrycollector" . | nindent 4 }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+spec:
+  mode: statefulset
+  replicas: 3
```

Comment:
maybe it is worthy to make this configurable, WDYT?

Reply:
There's a lot of settings that should be made configurable before we move this to a user-facing beta. For this PR, I wanted to limit the complexity, since it's big and complex as-is. So I only exposed the configuration necessary to run integration tests successfully.